swap/src/cli/event_loop.rs

```diff
@@ -167,12 +168,9 @@ impl EventLoop {
                             tracing::info!("Successfully closed connection to Alice");
                             return;
                         }
-                        SwarmEvent::UnreachableAddr { peer_id, address, attempts_remaining, error } if peer_id == self.alice_peer_id && attempts_remaining == 0 => {
-                            tracing::warn!(%address, "Failed to dial Alice: {}", error);
+                        SwarmEvent::OutgoingConnectionError { peer_id,  error } if matches!(peer_id, Some(alice_peer_id) if alice_peer_id == self.alice_peer_id) => {
+                            tracing::warn!( "Failed to dial Alice: {}", error);
 
-                            if let Some(duration) = self.swarm.behaviour_mut().redial.until_next_redial() {
```

Comment:
Why was this removed?

Reply:
this was a mistake thank you for catching it

Reply:
Ah great